### DISCUSSION TITLE
Docusaurus run with multiple languages

### DISCUSSION THREAD

**Discussion by MichaelStather** [facebook/docusaurus#7950] [Q&A]
AFAIK I can use either:

- `Docusaurus run` with automatical update on changes, but only for the default language
- `Docusaurus build` + `Docusaurus serve` without automatical update on changes, but all languages support

Now I need both, automatical  update on changes and all languages supported.

Is this possible?

**Comment by slorber** [comment]
Unfortunately, this is not supported, and you'll be able to find issues where this problem was already reported.

Each Docusaurus locale is its own unique single-page-application, it's not one APA for all languages at once. 

To keep DX better you also wouldn't want to run everything at once: it would become slower and slower to start as you add more locales to your config.
